hdwi/parser.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Hindawi
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let path   = parsedUrl.pathname;
+  // uncomment this line if you need parameters
+  let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  // console.error(parsedUrl);
+
+  let match;
+
+  if ((match = /^\/journals\/([a-z]+)\/([0-9]{4})\/([0-9]+)\/$/i.exec(path)) !== null) {
+    // https://www.hindawi.com/journals/apec/2021/5582774/
+    result.rtype    = 'ARTICLE';
+    result.mime     = 'HTML';
+    result.title_id = match[1];
+    result.publication_date = match[2]
+    result.unitid = match[3];
+    result.doi = match[3];
```

Comment:
This ID is not a DOI, after investigations on the platform, it seems that the DOI is : ``10.1155/2021/5582774``
Please add a DOI prefix (``const doiPrefix = '10.1155';``) as explained on a previous parser, modify the analogist analysis.

Example : 
![image](https://user-images.githubusercontent.com/38247370/143056279-fbce8646-6e20-495d-854e-d005df55ddd4.png)